### PR TITLE
Update Group section in UG

### PR DESCRIPTION
**Pull request by Bearypop**

### CODE REVIEW

**Review by wnchan — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -318,11 +318,18 @@ Deletes a specific student and all personal details based on email.
 
 ## Group Commands
 
+<div markdown="span" class="alert alert-primary">:exclamation: **Important:**
```
Comment (PearlynnT) [inline docs/UserGuide.md:321]: Thanks for adding the clarification

**Review by maj0-0 — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -318,11 +318,18 @@ Deletes a specific student and all personal details based on email.
...
+StudentConnect provides a set of group formation features to make it more convenient for students to find group mates. A group consists of 3 main components: a group number, a tutorial number and the group members.
+<br> Group number: Used to uniquely identify the group.
+<br> Tutorial number: This number is set by the creator of the group and serves as an indication of which tutorial class the members of the group are interested in enrolling in. It **does not** restrict students who do not have a matching tutorial number in their profile from joining the group.
```
[the remark this answers is not in the document]
Comment (neyapraveen) [reply docs/UserGuide.md:324]: Maybe can also add a link to the feature when mentioning it if possible

**Review by neyapraveen — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Bearypop) [reply docs/UserGuide.md:324]: Okay

**Pull request merged**